declarative-examples/simple-examples/credentialsMixedEnvironmentand.groovy

```diff
@@ -0,0 +1,28 @@
+pipeline {
+  environment {
+    // environment variables and credential retrieval can be interspersed
+    SOME_VAR = "SOME VALUE"
+    CRED1 = credentials("cred1")
+    INBETWEEN = "Something in between"
+    CRED2 = credentials("cred2")
+    OTHER_VAR = "OTHER VALUE"
```

Comment:
If it's possible for variable declarations to reference other vars in the value, such as `OTHER_VAR = "$CRED1 $CRED2"` that should be shown. Don't know whether this exists though.